explore/src/main/scala/explore/observationtree/AsterismGroupObsList.scala

```diff
@@ -313,7 +313,7 @@ object AsterismGroupObsList:
           val list        = obsIds.toList
           val div: TagMod = <.div(
             <.div(
-              list.toTagMod(id => <.div(id.show))
+              list.toTagMod(using id => <.div(id.show))
```

Comment:
It is strange that these methods take a function as a implicit. I'll send a PR to scalajs-react it ha fix

Reply:
The idea of the implicit is that if you pass a string or a component for example, it will convert automatically to vdom; but you can override like we do here.